pkg/terminal/proxy.go

```diff
@@ -181,13 +181,23 @@ func (p *Proxy) HandleProxyEnabled(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	enabled, err := workspaceOperatorIsRunning()
+	isOperatorInstalled, err := operatorIsInstalled()
 	if err != nil {
-		klog.Errorf("Failed to check if workspace operator is running: %s", err)
+		klog.Errorf("Failed to check if the web-terminal operator is installed: %s", err)
```

Comment:
After I modified manually console cluster role, I tried to run bridge and got error:
```
E0311 14:08:29.546308   23204 proxy.go:186] Failed to check if the web-terminal operator is installed: unknown
```

When I investigated more it leaded to:
```
responseBody: {"kind":"Status","apiVersion":"v1","metadata":{},"status":"Failure","message":"operators.operators.coreos.com \"web-terminal.openshift-operators\" is forbidden: User \"system:serviceaccount:openshift-console:console\" cannot get resource \"operators\" in API group \"operators.coreos.com\" at the cluster scope","reason":"Forbidden","details":{"name":"web-terminal.openshift-operators","group":"operators.coreos.com","kind":"operators"},"code":403}
error: unknown
```

So, I would avoid using Rest client not to handle response manually. As alternative I can propose dynamic client, I'll share proposal a bit later today

Reply:
When not found, error is logged into logs, while it's not exceptional case:

```
E0311 14:27:46.292581   27133 proxy.go:186] Failed to check if the web-terminal operator is installed: the server could not find the requested resource
```

Reply:
All of the above seems to be fixed with the new changes